### DISCUSSION TITLE
How to get the texture coordinates (uv) of each character in the osg:: Text object?

### DISCUSSION THREAD

**Discussion by RikiLee** [openscenegraph/OpenSceneGraph#1195] [Q&A]
Currently, I use this method to get the uv of each character :
`osgText::Text* text; `
`auto st = text->getShaderTechnique();`
`for (uint32_t  j = 0; j < glys.size(); ++j) {`
`    const osgText::Glyph::TextureInfo* tinfo = glys[j]->getTextureInfo(st);`
`    //tinfo->minTexCoord, tinfo->maxTexCoord; get the min and max uv`
`}`

but it makes the adjacent characters overlap

**Comment by robertosfield** [comment]
What do you need this info for?

The overlap of quads used for rendering is required to handle the rendering of any outline or shadow that is required and is perfectly normal.  The fragment shader discards any fragments with an alpha of 0.
